source/_examples/notify.mqtt.markdown

```diff
@@ -6,7 +6,15 @@ ha_category: Notifications
 ha_iot_class: Configurable
 ---
 
-The MQTT notification support is different than the other [notification](/integrations/notify/) platforms. It is a service. This means that you don't have to create a configuration entry but you need to provide more details when calling the service.
+The MQTT notification support is different than the other [notification](/integrations/notify/) platforms. It is a service.
+
+## Use the the MQTT notify platform
+
+The [MQTT notify service platform](/integrations/notify.mqtt/) allows you to use [MQTT discovery](/docs/mqtt/discovery/), or manual setup through `configuration.yaml` to enable a custom MQTT notify service. The notify platform helps to implement notification services on IoT notification screens, notification gateways, or for digital signage applications. An example is showing a text alert message on a climate control unit, e.g. an error, alert or informal message. The device can advertize the notification message service through discovery.
```

Comment:
Based on the code PR, it seems like only setup through discovery is supported (there's no `async_setup_platform`)?